src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-password-form/user-password-form.component.html

```diff
@@ -0,0 +1,117 @@
+<div class="col-sm-12 col-lg-6">
+  <form class="form-horizontal"
+        #frm="ngForm"
+        [formGroup]="userForm"
+        novalidate>
+    <div class="panel panel-default">
```

Comment:
panel has been dropped in bootstrap 4, we need to use the card class.
Please check `rgw-bucket-form.component.html` on how that is implemented. 
You also need to change the header, body and footers.
```suggestion
    <div class="card">
```